content/en/docs/setup/install/operator/index.md

```diff
@@ -75,11 +73,197 @@ accessible to `istioctl` by using this command:
+|---------|------------|
+Base | CRDs
+Traffic Management | Pilot
+Policy | Policy
+Telemetry | Telemetry
+Security | Citadel
+Security | Node agent
+Security | Cert manager
+Configuration management | Galley
+Gateways | Ingress gateway
+Gateways | Egress gateway
+AutoInjection | Sidecar injector
+
+In addition to the core Istio components, third-party addon features and components are also available:
+
+| Feature | Components |
+|---------|------------|
+Telemetry | Prometheus
+Telemetry | Prometheus Operator
+Telemetry | Grafana
+Telemetry | Kiali
+Telemetry | Tracing
+ThirdParty | CNI
+
+Features can be enabled or disabled, which enables or disables all of the components that are a part of the feature.
+Namespaces that components are installed into can be set at the global, feature, or component level, in increasing
+order of override priority.
+
+### Configure the feature or component settings
+
+After you identify the name of the feature or component from the previous table, you can use the API to set the values
+using the `--set` flag, or create an overlay file and pass it with the `--filename` flag. In general, the `--set` flag
+works well for customizing a few parameters, while overlay files are better for more extensive customization, or
+tracking configuration changes.
+
+The simplest customization is to turn a feature or component on or off from the configuration profile default. 
+
+For example, to disable the telemetry feature in a default configuration installation, use this command:
+
+{{< text bash >}}
+$ istioctl experimental manifest apply --set telemetry.enabled=false
+{{< /text >}}
+
+Alternatively, you can disable the telemetry feature using a configuration overlay file:  
+
+1. Create this file with the name telemetry_off.yaml and these contents:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  telemetry:
+    enabled: false
+{{< /text >}}
+
+1. Provide the telemetry_off.yaml overlay file to `manifest apply` command:
 
 {{< text bash >}}
-$ istioctl experimental profile list -s installPackagePath=https://github.com/istio/istio/releases/tags/1.3.3
+$ istioctl experimental manifest apply -f telemetry_off.yaml
+{{< /text >}}
+
+You can also use this approach to set the component-level configuration, such as enabling the node agent:
+
+{{< text bash >}}
+$ istioctl experimental manifest apply --set security.components.nodeAgent.enabled=true
+{{< /text >}}
+
+Another common customization is to select different namespaces for features and components. The following is an example
+of installation namespace customization:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  defaultNamespace: istio-system
+  security:
+    namespace: istio-security
+    components:
+      citadel:
+        namespace: istio-citadel
+{{< /text >}}
+
+Applying this file will cause the default profile to be applied, with components being installed into the following
```

Comment:
This part is overly complicated.  I'd try to rework as follows:

Some language about the action and the outcome. (152-153, 170-172).

1. Create this example `IstioControlPlane` resource:
cut and paste lines 157-165.
1. Apply the file using the following command:
insert command
1. profit!

Reply:
I reorganized a bit - your line numbers have changed so I'm not sure if this is what you mean but ptal.